elyra/metadata/schemas/runtime-image.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Schema for Runtime Image metadata",
+  "name": "runtime-image",
+  "properties": {
+    "schema_name": {
+      "type": "string",
+      "pattern": "^[a-z][a-z0-9-_]*[a-z0-9]$",
+      "minLength": 1
+    },
+    "display_name": {
+      "description": "The display name of the Runtime Image",
+      "type": "string",
+      "pattern": "^[a-zA-Z][a-zA-Z0-9-_. (){}]*[a-zA-Z0-9)}]$"
+    },
+    "metadata": {
+      "description": "Additional data specific to this Runtime Image",
+      "type": "object",
+      "properties": {
```

Comment:
We should add an optional [description property](https://github.com/elyra-ai/elyra/blob/master/elyra/metadata/schemas/code-snippet.json#L20-L23) to allow users to better describe their image.